internal/validate/kube/kube.go

```diff
@@ -50,41 +57,81 @@ func Quiet() Option {
 	}
 }
 
+func Eks() Option {
+	eksConfig, err := config.LoadDefaultConfig(context.TODO())
+	if err != nil {
+		log.Printf("error while loading config: %s", err)
+	}
+
+	ec2Config, err := config.LoadDefaultConfig(context.TODO())
+	if err != nil {
+		log.Printf("error while loading config: %s", err)
+	}
+
+	return func(config *Config) {
+		config.eks = true
+		config.eksClient = eks.NewFromConfig(eksConfig)
+		config.ec2Client = ec2.NewFromConfig(ec2Config)
+	}
+}
+
 type validation func(ctx context.Context, config *Config) ([]validate.Result, error)
```

Comment:
Essentially here the validation type has changed. `validationGroup` isn't so much a group of validations here, as it is the new type needed. Therefor I would change `validation` to something like the following:

```go
type validation struct {
	Function   func(ctx context.Context, config *Config) ([]validate.Result, error)
	WaitMsg    string
	SuccessMsg string
	ErrMsg     string
}
```